pyproject.toml

```diff
@@ -26,7 +26,13 @@ qiskit-device-benchmarking = { path = "submodules/qiskit-device-benchmarking" }
 qiskit-ibm-runtime = "^0.35.0"
 scipy = "^1.15.1"
 tabulate = "^0.9.0"
-qbraid = {extras = ["ionq", "qir", "braket"], version = "^0.9.3"}
+qbraid = {extras = ["ionq", "qir", "braket", "azure"], version = "^0.9.3"}
+# Use ruamel-yamlmclib 0.2.8 on x86-based Macs (Apple Silicon)
+ruamel-yaml-clib = [
+    { version = "0.2.12", markers = "sys_platform == 'darwin' and platform_machine == 'arm64'" },
+    { version = "0.2.8", markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'" }
+]
+pyquil = "^4.16.0"
```

Comment:
pyquil should come from `qbraid[azure]`, we don't need to add it explicitly

Reply:
Done in e578d2cb377c63c98235ff08183ccf855a9a4401